src/kOS/Binding/FlightControlManager.cs

```diff
@@ -225,6 +227,15 @@ public void SelectAutopilotMode(VesselAutopilot.AutopilotMode autopilotMode)
             }
         }
 
+        public string GetAutopilotModeName()
+        {
+            // TODO: As of KSP 1.0.4, RadialIn and RadialOut are swapped.  Check if still true in future versions.
```

Comment:
I tested it exactly on 1.1.2. And these are still inverted.
Don't know what and where they fixed, but apparently not this.